code/modules/power/lighting/light.dm

```diff
@@ -472,7 +472,7 @@
 	if(!has_emergency_power(power_usage_amount))
 		return FALSE
 	var/obj/item/stock_parts/cell/real_cell = get_cell()
-	if(real_cell.charge > 300) //it's meant to handle 120 W, ya doofus
+	if(real_cell.charge > 3e5) //it's meant to handle 120 W, ya doofus
```

Comment:
```suggestion
	if(real_cell.charge > 0.3 MEGA JOULES)
```

Reply:
I made it be based on the initial max charge of the emergency light fixture cell class.